docs/DeveloperGuide.md

```diff
@@ -765,23 +765,94 @@ testers are expected to do more *exploratory* testing.
+
+    1. Test case: `vcreate n/Ben p/98765432 e/ben@example.com s/chef`<br>
+       Expected: Volunteer is not created as there already exists a 'Ben' in the volunteer list. Error details shown in the status message.
+
+    1. Other incorrect delete commands to try: `vcreate`, `vcreate n/`, `...` <br>
+       Expected: `Invalid commat format!` error message shown in the status message. Instructions on the correct format are shown as well.
+
 ### Deleting a volunteer
 
-1. Deleting a volunteer while all volunteers are being shown
+1. Prerequisites: List all volunteers using the `vlist` command. Multiple volunteers in the list.
 
-   1. Prerequisites: List all volunteers using the `list` command. Multiple volunteers in the list.
+    1. Test case: `vdelete 1`<br>
+       Expected: First volunteer is deleted from the list. Details of the deleted volunteer shown in the status message.
 
-   1. Test case: `delete 1`<br>
-      Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message. Timestamp in the status bar is updated.
+    1. Test case: `vdelete 0`<br>
+       Expected: No volunteer is deleted. Error details shown in the status message.
 
-   1. Test case: `delete 0`<br>
-      Expected: No volunteer is deleted. Error details shown in the status message. Status bar remains the same.
+    1. Other incorrect delete commands to try: `vdelete`, `vdelete x`, `...` (where x is larger than the list size or a non-integer)<br>
+       Expected: Similar to previous.
 
-   1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
-      Expected: Similar to previous.
+### Finding a volunteer
 
-1. _{ more test cases …​ }_
+1. Prerequisites: List all volunteers using the `vlist` command. Multiple volunteers in the list.
+
+    1. Test case: `vfind n/George s/chef`<br>
+       Expected: Volunteer list is updated to show `George`, who is a `chef`. Number of matching volunteers shown in the status message.
+   
+    1. Test case: `vfind s/amazing`<br>
+       Expected: Volunteer list displays nothing. `0 volunteers listed!` shown in the status message.
+
+    1. Test case: `vfind n/!@#`<br>
+       Expected: Volunteer list displays nothing. `Names should only contain alphanumeric characters and spaces, and it should not be blank` shown in the status message.
+
+    1. Other incorrect delete commands to try: `vfind`, `vfind randomwordshere n/ben`, `...` <br>
+       Expected: `Invalid commat format!` error message shown in the status message. Instructions on the correct format are shown as well.
+
+### Finding an event
+
+
+1. Prerequisites: List all events using the `elist` command. Multiple events in the list.
+
+    1. Test case: `efind n/fixing computers`<br>
+       Expected: First event is deleted from the list..
```

Comment:
wait why event is deleted in efind test case?

Reply:
oops